src/economy/economy.cpp

```diff
@@ -3981,17 +3981,15 @@ float estimate_war_subsidies_income(sys::state& state, dcon::nation_id n) {
 }
 float estimate_reparations_income(sys::state& state, dcon::nation_id n) {
 	float total = 0.0f;
-	if(state.current_date < state.world.nation_get_reparations_until(n)) {
-		for(auto uni : state.world.nation_get_unilateral_relationship_as_target(n)) {
-			if(uni.get_reparations()) {
-				auto source = uni.get_source();
-				auto const tax_eff = nations::tax_efficiency(state, n);
-				auto total_tax_base = state.world.nation_get_total_rich_income(source) +
-					state.world.nation_get_total_middle_income(source) +
-					state.world.nation_get_total_poor_income(source);
-				auto payout = total_tax_base * tax_eff * state.defines.reparations_tax_hit;
-				total += payout;
-			}
+	for(auto uni : state.world.nation_get_unilateral_relationship_as_target(n)) {
```

Comment:
you should double check the loop that makes the payments too

Reply:
the loop that makes the payments is OK, it was just the estimation FOR INCOME that was wrong
assumedly by copy pasting code without changing a bit of the behaviour first